configuration/pih/htmlforms/hiv/section-hiv-history.xml

```diff
@@ -1530,11 +1530,13 @@
                                              answerConceptIds="CIEL:159943, CIEL:159944" />
                                     </p>
                                 </td>
-                                <td class="{drugname}-hx">
-                                    <obs conceptId="CIEL:1190" class="startDate"/>
+                                 <td class="{drugname}-hx">
+                                    <obs conceptId="CIEL:163526" class="startDate" allowFutureDates="true" allowPastDates="true"/>
                                 </td>
+                               
                                 <td class="{drugname}-hx">
-                                    <obs conceptId="CIEL:1191" class="endDate"/>
```

Comment:
@louidorjp Which concept do you want here?  Don't you want to keep the concept id so you don't need to migrate data?

Reply:
Yes @lnball , I kept the existing concepts